src/actions/eventListeners.js

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { getEventListener, getEventListeners } from '../api';
+import {
+  fetchNamespacedCollection,
+  fetchNamespacedResource
+} from './actionCreators';
+
+export function fetchEventListener({ name, namespace }) {
+  return fetchNamespacedResource('EventListener', getEventListener, {
+    name,
+    namespace
+  });
+}
+
+export function fetchEventListeners({ namespace } = {}) {
+  return fetchNamespacedCollection('EventListener', getEventListeners, {
+    namespace
+  });
```

Comment:
Should this include `filters` in the parameter object to be consistent with `triggerBindings.js` and the `getEventListeners()` function?

If so, we should also add `filters` to the test (see following comment about this for `triggerBindings.test.js`).

Reply:
Done and done